lib/lightning/janitor.ex

```diff
@@ -40,9 +40,15 @@ defmodule Lightning.Janitor do
       stream
       |> Stream.each(fn run ->
         Runs.mark_run_lost(run)
+
+        run
+        |> Repo.preload([:log_lines, work_order: [:workflow]])
+        |> Lightning.FailureAlerter.alert_on_failure()
       end)
       |> Stream.run()
 
+      # TODO - this appears to be a duplicate of https://github.com/OpenFn/lightning/blob/main/lib/lightning/runs.ex#L291
+      # but without it, the tests don't pass. @midigofrank, do you have ideas?
```

Comment:
Thank God the tests are failing. Let me check it out

Reply:
Hey @midigofrank , I assume we have a release tomorrow AM. Did you check it out? Let me know if not so I can work on it and merge before the release is cut.

Reply:
This here is the failing test right? We should just inverse it because at this moment, it is highly unlikely that we will find a `lost run` that still has `running steps`
```sh
  3) test find_and_update_lost/0 updates steps whose run has finished but step hasn't (Lightning.JanitorTest)
     test/lightning/janitor_test.exs:108
     Expected truthy, got false
     code: assert is_struct(reloaded_step.finished_at, DateTime)
       test/lightning/janitor_test.exs:180: (test)
```